infrastructure/stacks/service_etl/functions/service_etl_sns/service_etl_sns.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at## http://aws.amazon.com/apache2.0/
+# or in the "license" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+# Description: This Lambda function sends an email notification to a given AWS SNS topic when a particular
+#              pattern is matched in the logs of a selected Lambda function. The email subject is
+#              Execution error for Lambda-<insert Lambda function name>.
+#              The JSON message body of the SNS notification contains the full event details.
+
+# Author: Sudhanshu Malhotra
+
+import base64
+import boto3
+import gzip
+import json
+import logging
+import os
+
+from botocore.exceptions import ClientError
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def logpayload(event):
+    logger.setLevel(logging.DEBUG)
```

Comment:
Do we need this down to the DEBUG level here?  Also, should we create a couple of logger.info lines so we can see if this is fired when we are running at INFO level?